app/controllers/v0/sign_in_controller.rb

```diff
@@ -225,6 +228,46 @@ def introspect
       render json: { errors: e }, status: :unauthorized
     end
 
+    def fetch_openid_configuration
+      render json: openid_configuration_data
+    end
+
+    def openid_configuration_data
+      openid_data.merge(openid_data_two)
+    end
+
+    def openid_data
+      {
+        issuer: 'https://www.rfc-editor.org',
```

Comment:
None of these values are accurate. This appears to be a .well-known for the `rfc-editor.org` page, not for `va.gov`